artifact-registry/terraform/environments/prod/variables.tf

```diff
@@ -0,0 +1,71 @@
+###################################
+# Provider related values
+###################################
+
+variable "gcp_region" {
+  type        = string
+  default     = "europe-west4"
+  description = "Default Google Cloud region to create resources."
+}
+
+variable "gcp_project_id" {
+  type        = string
+  default     = "sap-kyma-prow"
+  description = "Google Cloud project to create resources."
+}
+
```

Comment:
Non root modules should not have provider blocks. These variables are not needed.

Reply:
`│ Error: Reference to undeclared input variable
│ 
│   on ../../../../configs/terraform/modules/artifact-registry/main.tf line 5, in resource "google_artifact_registry_repository" "artifact_registry":
│    5:   location      = var.artifact_registry_multi_region == true ? var.artifact_registry_primary_area : var.gcp_region
│ 
│ An input variable with the name "gcp_region" has not been declared. This variable can be declared with a variable "gcp_region" {} block.`